src/sdam/public.rs

```diff
@@ -133,3 +173,125 @@ impl<'a> fmt::Display for ServerInfo<'a> {
         write!(f, " }}")
     }
 }
+
+/// A description of the most up-to-date information known about a topology. Further details can
+/// be found in the [Server Discovery and Monitoring specification](https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst).
+#[derive(Clone)]
+pub struct TopologyInfo {
```

Comment:
(continuing the conversation here since GitHub won't let me comment on the old thread, cc @abr-egn)

It does seem a bit odd to name this struct `TopologyInfo` but have its only use be as an aliased type. Maybe a better middle ground would be to name this `EventTopologyDescription` and remove the alias on it and keep the alias on `ServerInfo` but update it to `EventServerDescription`. That way the number of names for each type corresponds to the number of use-cases for each. As part of that we'd need to move this definition to `src/event` to live with the alias I guess. What do you all think?

Reply:
Sounds reasonable to me.